src/color/translators.ts

```diff
@@ -156,16 +163,17 @@ export const hueRYB = (hue: number, toRYB: boolean): number => {
     let c = 0;
     let d = 0;
 
-    from.find((arr: [number, number], index: number): boolean => {
-        if (hue >= arr[0] && hue < arr[1]) {
-            a = arr[0];
-            b = arr[1];
-            c = to[index][0];
-            d = to[index][1];
-            return true;
+    for (let i = 0; i < from.length; i++) {
```

Comment:
What is the purpose of this change? If it is for legibility, I prefer the previous one.